vmaas/cache.go

```diff
@@ -25,8 +25,9 @@ type Cache struct {
 
 	ArchCompat map[ArchID]map[ArchID]bool
 
-	PackageDetails map[PkgID]PackageDetail
-	Nevra2PkgID    map[Nevra]PkgID
+	PackageDetails    map[PkgID]PackageDetail
+	Nevra2PkgID       map[Nevra]PkgID
+	NameID2SrcNameIDs map[NameID]map[NameID]struct{}
```

Comment:
it can't be `map[NameID]NameID` because many packages names have multiple source packages, e.g. `katello-agent` can be built from `katello-agent` or `katello-host-tools` or `kernel-rt-core` can be built from `kernel-rt` or `kernel`